lib/htsget-server/docker-compose.yml

```diff
@@ -38,7 +33,14 @@ services:
         target: secret
       - source: selfsigned-site-pem
         target: cert.pem
+      - source: vault-s3-token
+        target: vault-s3-token
+      - source: candig-opa-secret
+        target: candig-opa-secret
     logging: *default-logging
     environment:
       HTSGET_TEST_KEY: "hoodlebug"
+      OPA_URL: ${OPA_URL}
+      CANDIG_AUTH: ${CANDIG_AUTHORIZATION}
```

Comment:
What is the CANDIG_AUTHORIZATION field used for?

Reply:
It's a flag that lets you choose whether or not the auth mechanism is OPA or something else